test-time_slot-server/.env

```diff
@@ -0,0 +1,5 @@
+CONNECTION_STRING = postgres://dngcgyzv:aZtRAwtsTxmLk-HHzdcHlYO6vtgO9TYj@queenie.db.elephantsql.com:5432/dngcgyzv
```

Comment:
Env variable should be in gitignore.
No tokens, secrets and password should be exposed in repo.